tests/erlang_tests/test_integer_to_binary.erl

```diff
@@ -46,3 +53,27 @@ compare_bin(Bin1, Bin2, Index) ->
         _Any ->
             0
     end.
+
+test_bases() ->
+    <<"0">> = integer_to_binary(0),
+    <<"0">> = erlang:integer_to_binary(0, 10),
+    <<"-1">> = integer_to_binary(-1),
+    <<"-1010">> = erlang:integer_to_binary(-10, 2),
+    <<"1010">> = erlang:integer_to_binary(10, 2),
+    <<"A">> = erlang:integer_to_binary(10, 16),
+    <<"123456789ABCDEF0">> = erlang:integer_to_binary(16#123456789ABCDEF0, 16),
+    <<"A">> = erlang:integer_to_binary(10, 36),
+    assert_badarg(fun() -> erlang:integer_to_list(10, 1) end),
+    assert_badarg(fun() -> erlang:integer_to_list(10, 0) end),
+    assert_badarg(fun() -> erlang:integer_to_list(10, -1) end),
+    assert_badarg(fun() -> erlang:integer_to_list(10, 37) end),
```

Comment:
I think that this should be `erlang:integer_to_binary`.
Also, does `erlang:integer_to_list` gets compiled to an actual call to `erlang:integer_to_list`, or does it gets replaced with a constant?